src/components/radio/Radio.stories.mdx

```diff
@@ -1,32 +1,53 @@
 import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
-import Radio from './Radio';
+import Radio from './RadioGroup';
+import RadioButton from './RadioButton';
+import Icon from 'components/icon/Icon';
 
 <Meta title="Forms|Radio" component={Radio} />
 
 # Radio
 
-Our radio buttons are defined as a group. Use this over checkboxes when you only want to provide one choice to the user. For now you can only input text as labels, not stuff like icons.
+The radio group is created by passing an `Array` of choices as: 
```

Comment:
This is a pretty unstandard way of creating radio-groups. But I don't really mind it.

Reply:
Hm, I agree. I have iterated a bit through different ways of doing it and ended up with this one.
In retrospect, I think I will see if it can rather be done through `React.cloneElement()` so that the structure can be 
```js
<RadioGroup onChange={onChange} groupName={groupName}>
  <RadioButton value={value}>{label}</RadioButton>
  <RadioButton value={value} checked>{label}</RadioButton>
</RadioGroup>
```
With `onChange`, `groupName` and `checked` being optional